### ISSUE TITLE
Iframe Authentication Redirection

### ISSUE THREAD

**Issue by Nenillo** [RocketChat/Rocket.Chat#18235]
### Description:

We are using iframe authentication, on the iframe we show a button so the user can go to our website, use the different login options and once the user is logged in is redirected back to RocketChat. At this point RocketChat uses the SSO endpoint to get the login information and the user gets logged into the chat. This works properly on the browser but not on the mobile application because it loads the RocketChat web version inside the webview.

### Steps to reproduce:

1. Implement Iframe Authentication
2. Redirect the user to your website, not within the iframe, would be like a target="_top" link
3. Log the use rin
4. Go back to RocketChat

### Expected behavior:

In mobile, after going back to the RocketChat url, close the webview and load the mobile app

### Actual behavior:

In mobile, after going back to the RocketChat url, the RocketChat web app is loaded in the webview.

### Server Setup Information:

- Version of Rocket.Chat Server: 3.4.1
- Operating System: Linux 3.13
- Deployment Method: docker image
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: v12.16.1
- MongoDB Version: 4.0.18

### Client Setup Information

- Desktop App or Browser Version: Chromium 83.0.4103.116
- Operating System: Ubuntu 20.04

**Comment by aasseman** [comment]
Having the same problem with GSuite SAML. Having the SAML login open in the Rocket.Chat app prevents us from using FIDO 2FA.

**Comment by didacroyo** [comment]
Same problem also with the newest versions:
Rocket.Chat Server: 3.4.2 
Mobile App (Android) with version 4.8.1.2131

**Comment by PhilippVerpoort** [comment]
I think I'm seeing a similar problem with SAML in the Android app. After authenticating on the website of the SimpleSAML IdP, I'm logged into my rocket.chat but the login remains inside the browser window, not the actual app. So after closing the window, the client app is still not logged in.

Rocker.Chat Server: 3.2.2 (installed via snap packages)
Mobile App (Android) with version 3.x
--- 
And as I was testing this, I noticed my Android app was outdated, so I upgraded to 4.8.1.2131 and now it seems to be working.

I'll post again if the problem comes up again.
